test/python/test_api.py

```diff
@@ -495,6 +495,75 @@ def test_substitute(env):
     assert terms == [P_a, P_b]
 
 
+def test_timeout(env):
+    """
+    Test to validate that it possible to set a timeout via the API
+    """
+
+    # Constant for the internal Bitwuzla value for when CaDiCaL is enabled
+    CADICAL_SOLVER_ID = 4
+
+    # Try enabling CaDiCaL
+    env.bzla.set_option(Option.SAT_ENGINE, "cadical")
+
+    # Check if CaDiCaL has been enabled
+    if env.bzla.get_option(Option.SAT_ENGINE) != CADICAL_SOLVER_ID:
```

Comment:
```suggestion
    if env.bzla.get_option(Option.SAT_ENGINE) != "cadical":
```
This is now possible since e1d22fed1707511ab55a4add939eb1545bab0166